### DISCUSSION TITLE
VisDrone Dataset

### DISCUSSION THREAD

**Discussion by khaghanijavad** [STVIR/pysot#295] [General] [answered]
Hi, 

Have you tested the Siamese networks contained in pysot on VisDrone dataset? If you have tried this, could you please share the results? 

Thanks.

**Comment by ZhiyuanChen** [comment]
If it is not listed, I believe we didn't. 
However, you are more than welcome to test it and add the results thorugh a pull request

**Comment by khaghanijavad** [comment]
Thank you very much. By the way, I can only find SiamMask, SiamRPN++, and SiamRPN++ LongTerm in the trackers folder. Have you also included DaSiamRPN, SiamRPN, SiamFC, and maybe other Siamese networks in Pysot?

**Comment by ZhiyuanChen** [comment, marked as answer]
> Thank you very much. By the way, I can only find SiamMask, SiamRPN++, and SiamRPN++ LongTerm in the trackers folder. Have you also included DaSiamRPN, SiamRPN, SiamFC, and maybe other Siamese networks in Pysot?

I believe they are not yet provided...
